lib/customer.rb

```diff
@@ -0,0 +1,10 @@
+module HotelBooking
+  class Customer
+    attr_reader :id, :name
+
+    def initialize(id:, name:)
+      @id = id
+      @name = name
+    end
+  end
+end
```

Comment:
No tests! Maybe this is a result of realizing that our project doesn't require Customers being implemented, so it was hard to understand what to test a Customer to do :)